app/concepts/ticket/use_case/taken.rb

```diff
@@ -4,7 +4,8 @@ class Taken < UseCase::Base::Base
         def call(id, cinema_hall_id, movie_id)
           seats_taken=[]
           i = 0
-          repository.where(cinema_hall_id: cinema_hall_id, movie_id: movie_id).each do |ticket|
+          find = repository.where(cinema_hall_id: cinema_hall_id, movie_id: movie_id) rescue nil
```

Comment:
Shouldn't be sth like that?
```suggestion
          tickets = repository.where(cinema_hall_id: cinema_hall_id, movie_id: movie_id) rescue nil
```